java/pom.xml

```diff
@@ -392,6 +392,7 @@
                                     <arg value="-DGPU_ARCHS=${GPU_ARCHS}"/>
                                     <arg value="-DCUDF_JNI_ARROW_STATIC=${CUDF_JNI_ARROW_STATIC}"/>
                                     <arg value="-DCUDF_JNI_LIBCUDF_STATIC=${CUDF_JNI_LIBCUDF_STATIC}"/>
+                                    <arg value="-DBUILD_SHARED_LIBS=ON"/>
```

Comment:
Should this not use `${BUILD_SHARED_LIBS}`?

Reply:
No, because the Java bindings require a shared library in order to work properly.  Specifying `BUILD_SHARED_LIBS=OFF` here would result in a jar that cannot work.  Specifying `BUILD_SHARED_LIBS=OFF` only makes sense when not trying to build the cudf jar (i.e.: leveraging the code from an external project).

Reply:
> built as static libraries rather than shared libraries by specifying -DBUILD_SHARED_LIBS=OFF when configuring with cmake.

Ah, so. I should have read the description above more carefully.